ykrt/pass_finder/try_passes.py

```diff
@@ -43,7 +43,7 @@ def __str__(self):
         link_time = ",".join([ str(p) for p in self.link_time ])
         return f"PipelineConfig(pre_link=[{pre_link}], link_time=[{link_time}])"
 
-def get_pipeline_config(is_prelink, ok_passes, try_passes):
+def get_pipeline_config(is_prelink, try_passes, ok_passes=[]):
```

Comment:
We can't use `[]` is a default argument in Python (well, unless they've finally fixed it in Python 3?), because it's a singleton that's shared amongst all calls to the function. The standard idiom is:

```python
def f(x = None):
  if x = None: x = []
```

Reply:
They didn't fix this in Python 3:

```python
In [2]: def f(x=[]):
   ...:     x.append(1)
   ...:     print(x)
   ...: 

In [3]: f()
[1]

In [4]: f()
[1, 1]
```

It's OK as long as you don't mutate it, but I do tend to agree, that it's best to not pass lists at all.

Reply:
Addressed: 6c3f62f2dd481f0a792a6e681a1d4f0fe6b336fc